packages/paste-website/src/pages/experiences/console/alarm-setup/index.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: About Paste
```

Comment:
Issue: the meta data isn't correct for the page. It should  have the alarm setup info, not about paste.